### ISSUE TITLE
[5pt] Alaska waterbodies layer is assigned wrong projection

### ISSUE THREAD

**Issue by mluck** [NOAA-OWP/inundation-mapping#1142]
The Alaska waterbodies layer (`inputs/nwm_hydrofabric/nwm_waterbodies_alaska.gpkg`) is assigned an incorrect projection. It is assigned EPSG:3338 but the data are in geographic (lat/lon) coordinates
![image](https://github.com/NOAA-OWP/inundation-mapping/assets/6616694/24fff243-da53-4d55-8cb6-f42b683dc18c)

Other datasets in EPSG:3338 (e.g., catchments) are in projected units of meters
![image](https://github.com/NOAA-OWP/inundation-mapping/assets/6616694/02615984-4739-4bf4-ac2f-95add49d014b)

This projection issue causes no lakes to be returned in pre-clipping because `nwm_lakes` doesn't overlay with `wbd_buffer` (which is in EPSG:3338) here:
https://github.com/NOAA-OWP/inundation-mapping/blob/f30a48627e81c4169a1ed53e46811ba42dbabf2b/data/wbd/clip_vectors_to_wbd.py#L88

A simple solution would be to assign the correct projection and then reproject the data to EPSG:3338. We just need to figure out if the erroneous projection was present in the original data or if it was introduced in preprocessing. We should also check to make sure the other NWM Alaska layers are in the correct projection.

**Comment by RobHanna-NOAA** [comment]
I tried a simple preproject in QGIS and it failed, likely because it does not know it's original projection. We could assign it if we knew it in the first place. Need to go back to the person/source of where we got it.

**Comment by RobHanna-NOAA** [comment]
URGENT and will require re-running pre-clip for all Alaska HUCs as soon as possible (for this weeks release)

**Comment by mluck** [comment]
@EmilyDeardorff is on this and has figured out the original projection.

**Comment by EmilyDeardorff** [comment]
I've exported a version of `nwm_waterbodies_alaska.gpkg` with the corrected projection and a note in the geopackage metadata indicating this change. 

I've uploaded the updated file to the shared drive in a temporary location (`/efs-drives/fim-dev-efs/fim-data/temp/emily/corrected_ak_waterbodies_may2024`). 

Once this file is reviewed, it can be copied into the `inputs/nwm_hydrofabric` directory across our processing environments.

**Comment by EmilyDeardorff** [comment]
I've exported a version of `nwm_waterbodies_alaska.gpkg` with the corrected projection and a note in the geopackage metadata indicating this change. 

I've uploaded the updated file to the shared drive in a temporary location (`/efs-drives/fim-dev-efs/fim-data/temp/emily/corrected_ak_waterbodies_may2024`). 

Once this file is reviewed, it can be copied into the `inputs/nwm_hydrofabric` directory across our processing environments.  

@mluck Would you review this new file?

Once it is approved, @RobHanna-NOAA can copy it to our other environments.

**Comment by RobHanna-NOAA** [comment]
Yup... The new one looks great. I layered it against the WBD in QGIS and it is fine. Go ahead and get a PR in for it and I can test fim pipeline on it.  Then I can re-run pre-clip for just the Alaska HUCs and get into the main FIM system.

**Comment by mluck** [comment]
@RobHanna-NOAA There's no code change so I'm not sure if we need a PR for this?

**Comment by RobHanna-NOAA** [comment]
oh ya... good point.  ok. I will do a quick pipeline test on it and update this and we can close it.  :)

**Comment by EmilyDeardorff** [comment]
Sounds good, thank you!
